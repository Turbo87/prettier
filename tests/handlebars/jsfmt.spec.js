run_spec(__dirname, { parser: "handlebars" });
