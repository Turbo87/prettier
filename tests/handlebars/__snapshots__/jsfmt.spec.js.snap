// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getting_started.hbs 1`] = `
<div class="entry">
    <h1>{{title}}</h1>
    <div class="body">
        {{body}}
    </div>
</div>~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
<div class="entry"><h1>{{title}}</h1><div class="body">{{body}}</div></div>
`;
